.github/workflows/integration_test_secrets.yaml

```diff
@@ -0,0 +1,79 @@
+name: Integration tests with secrets
+
+on:
+  workflow_call:
+
+env:
+  REGISTRY: ghcr.io
+  OWNER: ${{ github.repository_owner }}
+
+jobs:
+  get-runner-image:
+    name: Get runner image
+    uses: canonical/operator-workflows/.github/workflows/get_runner_image.yaml@main
+  build-images:
+    uses: canonical/operator-workflows/.github/workflows/build_images.yaml@main
+    needs: get-runner-image
+    with:
+      owner: ${{ github.repository_owner }}
+      registry: ghcr.io
+      runs-on: ${{ needs.get-runner-image.outputs.runs-on }}
+      trivy-image-config: trivy.yaml
```

Comment:
Couldn't we reuse the image built by the other "integration" test ?
We'll be building and testing the images twice

Reply:
I saw your change, great, I close the comment here 👍

Reply:
https://github.com/canonical/operator-workflows/pull/90